index.html

```diff
@@ -6,6 +6,7 @@
   <link rel="icon" type="image/svg+xml" href="/vite.svg" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Project 2</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```

Comment:
Temporary icons, material ui icons will be removed from the repository once we create our own:
https://team-voyagers.atlassian.net/browse/P2-181?atlOrigin=eyJpIjoiNDQwMjdmOGY0NzMzNDM4N2IxMzFkOGM4ZThiZTUyNGYiLCJwIjoiaiJ9